### DISCUSSION TITLE
ART 1.19.2

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2672] [Announcements]
This release of ART 1.19.2 provides updates to ART 1.19

# Added

[None]

# Changed

- Replaced `art/attacks/evasion/overload/box_iou.py` under APGL license with code released under MIT License (#2615)

# Removed

- Removed accidental installation of tests module in site-packages (#2616)

# Fixed

- Fixed bug in `PyTorchObjectDetector. _get_losses ` which did not freeze batch normalisation layers of the object detection model. All methods of `PyTorchObjectDetector` calculating losses or loss gradients therefore modified the batch normalisation parameters of the model. This resulted in inaccurate loss gradients and a gradually changing model in its batch normalisation layers. (#2663)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.19.2'>ART 1.19.2</a>.</em>
